ffi/Makefile

```diff
@@ -14,22 +14,31 @@ ifeq ($(OS),Windows_NT)
     endif
 	OS_EXT=dll
 else
-	UNAME_OS := $(shell uname -s)
-	ifeq ($(UNAME_OS),Darwin)
-		OS_FLAG=darwin
-	endif
-	ifeq ($(UNAME_OS),Linux)
-		OS_FLAG=linux
+	ifeq ($(strip $(OS_FLAG)),)
+		UNAME_OS := $(shell uname -s)
+		ifeq ($(UNAME_OS),Darwin)
```

Comment:
Is all this logic duplicating what go-releaser GH action does (in terms of deciding how to name artefacts)

Reply:
Well goreleaser doesnt seem configured for all of these - it only releases the linux_x86 binary so far by the looks of it.